cpp/core/CMakeLists.txt

```diff
@@ -325,7 +325,7 @@ else()
 endif()
 
 target_link_libraries(gluten
-    PUBLIC Arrow::arrow Arrow::parquet)
+    PUBLIC Arrow::arrow Arrow::parquet glog::glog)
```

Comment:
The target found by our `Findglog` module is google::glog.

Reply:
thank you!

Reply:
> The target found by our `Findglog` module is google::glog.

This change should be necessary.